components/raftstore/src/store/worker/region.rs

```diff
@@ -243,28 +248,79 @@ impl PendingDeleteRanges {
     }
 }
 
-#[derive(Clone)]
-struct SnapContext<EK, R>
+struct SnapGenContext<EK, R> {
+    engine: EK,
+    mgr: SnapManager,
+    router: R,
+}
+
+pub struct Runner<EK, R, T>
```

Comment:
let's not reorder it

Reply:
There's no reorder, the old `SnapContext` is removed.